ConceirgeDinning.Adapter.USRestaraunt/Translator/SupplierModelsToUiModelsTranslator.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+using ConceirgeDinning.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConceirgeDinning.Adapter.USRestaraunt.Translator
+{
+    class SupplierModelsToUiModelsTranslator
+    {
+        public static List<Restaurant> Translate(Models.SearchResponse response)
```

Comment:
Why you have made this method as static?

Reply:
It is our translator so our class need to be static so we have made static method by mistake i forget to write static keyword in class